keystone-init/keystone_init.py

```diff
@@ -733,9 +825,35 @@ def load_domains(ks, domains, member_role_name):
 
 
 def load_endpoints(ks, endpoints):
```

Comment:
Having that said, I'd simply rename this method to load_services.

Reply:
Done